examples/Demo/FluentUI.Demo.Client/Documentation/Components/Dialog/FluentDialog.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Dialog
+route: /Dialog
+---
+
+# Dialog
+
+`FluentDialog` is a window overlaid on either the primary window or another dialog window.
```

Comment:
This is a bit in contradiction with line 24 below. Maybe use the text from fluent2 site:

A dialog is a supplemental surface that can provide helpful interactions or require someone to take an action before they can continue their task, like confirming a deletion.

Dialogs are often interruptions, so use them for important actions. If you need to give someone an update on an action they just took but that they don't need to act on, try a toast.

Reply:
I copied the text from https://web-components.fluentui.dev/?path=/docs/components-dialog-dialog-body--docs
But make sense. I will change that.